website/static/js/nodeControl.js

```diff
@@ -234,23 +234,31 @@ var ProjectViewModel = function(data) {
     /**
      * Toggle the watch status for this project.
      */
+    var watchUpdateInProgress = false;
+    var noWatchUpdateInProgress = function (){
```

Comment:
Is this function necessary? I don't think it adds any more clarity than doing `!watchUpdateInProgress`.